Client/mods/deathmatch/logic/CEvents.cpp

```diff
@@ -37,16 +37,17 @@ bool CEvents::AddEvent(const char* szName, const char* szArguments, CLuaMain* pL
         if (pEvent->bAllowRemoteTrigger != bAllowRemoteTrigger)
             return false;
         
-        // Add pLuaMain to the set, std::unordered_set guarantees unique elements
-        pEvent->pLuaMainSet.insert(pLuaMain);
+        // Add pLuaMain to the vector, in case it's not already there
+        if (std::find(pEvent->pLuaMainVector.begin(), pEvent->pLuaMainVector.end(), pLuaMain) == pEvent->pLuaMainVector.end())
+            pEvent->pLuaMainVector.push_back(pLuaMain);
```

Comment:
Actually, sorry.. Now that I think about it, I think we have this `ListContains` method. Although stupidly named, I think it should work, please take a look at it.